src/Splunk.Logging.Common/HttpEventCollectorSender.cs

```diff
@@ -223,9 +232,21 @@ public enum SendMode
 
         private void DoSerialization(HttpEventCollectorEventInfo ei)
         {
+            
+            string serializedEventInfo;
+            if (formatter == null)
+            {
+                serializedEventInfo = SerializeEventInfo(ei);
+            }
+            else
+            {
+                var formattedEvent = formatter(ei);
+                ei.Event = formattedEvent;
+                serializedEventInfo = JsonConvert.SerializeObject(ei);
```

Comment:
Question for @glennblock: do we need the class level `JsonSerializer` if we use `JsonConvert` here? If yes, then do we need to use it here instead of `JsonConvert`?